java/Ice/greeter/client/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    java
+    application
+    id("com.zeroc.gradle.ice-builder.slice") version "1.5.1"
+}
+
+dependencies {
+    implementation("com.zeroc:ice:3.8.0-nightly-+")
+}
+
+slice {
```

Comment:
I assume we can include comments in these files?

Reply:
Yes I will add some comments here.